types/office-js/index.d.ts

```diff
@@ -4469,727 +4541,2433 @@ declare namespace Office {
+        */
+       loadCustomPropertiesAsync(callback: (result: AsyncResult) => void, userContext?: any): void;
+    }
+    export interface ItemCompose extends Item {
+        /**
+         * Gets or sets the description that appears in the subject field of an item.
+         * 
+         * The subject property gets or sets the entire subject of the item, as sent by the email server.
+         * 
+         * *Read mode*
+         * 
+         * The subject property returns a string. Use the normalizedSubject property to get the subject minus any leading prefixes such as RE: and FW:.
+         * 
+         * *Compose mode*
+         * 
+         * The subject property returns a Subject object that provides methods to get and set the subject.
+         * 
+         * [Api set: Mailbox 1.0]
+         * 
+         * @remarks
+         * 
+         * Minimum permission level: ReadItem
+         * 
+         * Applicable Outlook mode: Compose or read
+         */
+        subject: Subject;
+        /**
+         * Adds a file to a message or appointment as an attachment.
+         * 
+         * The addFileAttachmentAsync method uploads the file at the specified URI and attaches it to the item in the compose form.
+         * 
+         * You can subsequently use the identifier with the removeAttachmentAsync method to remove the attachment in the same session.
+         * 
+         * [Api set: Mailbox 1.1]
+         * 
+         * @remarks
+         * Minimum permission level: ReadWriteItem
+         * 
+         * Applicable Outlook mode: Compose
+         * 
+         * Errors: 
+         * 
+         * AttachmentSizeExceeded - The attachment is larger than allowed.
+         * 
+         * FileTypeNotSupported - The attachment has an extension that is not allowed.
+         * 
+         * NumberOfAttachmentsExceeded - The message or appointment has too many attachments.
+         * 
+         * @param uri The URI that provides the location of the file to attach to the message or appointment. The maximum length is 2048 characters.
+         * @param attachmentName The name of the attachment that is shown while the attachment is uploading. The maximum length is 255 characters.
+         * @param options Optional. An object literal that contains one or more of the following properties.
+         *        asyncContext: Developers can provide any object they wish to access in the callback method.
+         *        inInline: If true, indicates that the attachment will be shown inline in the message body, and should not be displayed in the attachment list.
+         * @param callback When the method completes, the function passed in the callback parameter is called with a single parameter, asyncResult, which is an AsyncResult object. On success, the attachment identifier will be provided in the asyncResult.value property. If uploading the attachment fails, the asyncResult object will contain an Error object that provides a description of the error.
```

Comment:
"a single parameter, asyncResult, which is an AsyncResult object"
"a single parameter of type AsyncResult"